cmds/core/uptime/uptime.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Get the time the machine has been up
+//
+// Synopsis:
+//     uptime
+//
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+)
+
+// This function takes in the filepath to a file containing loadavgs,it then extracts and returns the three load averages
+func loadavg(filepath string) (loadaverage string, err error) {
+
+	loadavg := strings.Fields(filepath)
+	loadavgstring := "loadaverage :" + loadavg[0] + "," + loadavg[1] + "," + loadavg[2]
```

Comment:
Don't return a string here either, but formatting would be like this:

fmt.Sprintf("loadaverage: %s", strings.Join(", ", loadavg[:2])